cluster-autoscaler/cloudprovider/oci/nodepools/oci_manager.go

```diff
@@ -634,6 +647,34 @@ func addTaintToSpec(node *apiv1.Node, taintKey string, effect apiv1.TaintEffect)
 	return true
 }
 
+func getEphemeralResourceRequestsInBytes(tags map[string]string) (int64, error) {
+	for key, value := range tags {
+		if key == npconsts.EphemeralStorageSize {
+			klog.V(0).Infof("ephemeral-storage size set with value : %v", value)
+			value = strings.ReplaceAll(value, " ", "")
+			valueLength := len(value)
+			if valueLength < 3 {
+				return 0, fmt.Errorf("invalid value set for ephemeral-storage requests")
+			}
+			unit := value[valueLength-3:]
+			if unit != "GiB" && unit != "MiB" {
+				return 0, fmt.Errorf("invalid value set for ephemeral-storage requests")
```

Comment:
If possible, we should make the error measurement easier for the user to know what to do.
"I put a wrong value, but the error doesn't indicate what the value should be"

This comment might be irrelevant if we use the kubernetes resource library.

Reply:
Thanks, yes, I have made the changes to use the k8s library.